### DISCUSSION TITLE
SSC SPORT 3

### DISCUSSION THREAD

**Discussion by Belal133** [iptv-org/community#506] [Show and tell]
### Channel Name

SSC SPORT 3

### Channel ID

SSC3.sa

### Stream URL

https://list.iptvcat.com/my_list/s/b17291a57f03b6f68cd24f7dbe96e747.m3u8

### Notes

Ssc sport 3
Ssc sport 4
Ssc sport 6
No signal 

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
Redirects to http://skylogic.site:8080/Unplugged83/F75DCxx397DMKop5388mM/280134, which is an Xtream-Codes server, we do not accept these kind of links in our playlists.

**Comment by R4IN2007** [comment]
SSC Sports 1 HD
https://shls-mbc1ksa-prod-dub.shahid.net/out/v1/6ae0a2ebab224c72ab9c298afeec8d91/index.m3u8

SSC Sports 2
https://ssc-6-ak.akamaized.net/out/v1/efbe58d3435d4de1beb1d7e3b27d309f/index.m3u8

SSC Sports 3
http://iptv.ipslow.com/tv31_www.elahmad.com_ssc_3/index.m3u8?token=fcc78993c540b17f402681987ae609c03d91f9c8-2ddbec0c99c9779c49a8b961581ef8df-1658993768-1658982968

SSC Sports 4
http://iptv.ipslow.com/tv31_www.elahmad.com_ssc_4/index.m3u8?token=60de0ba0aa3529cd2bfc576a0345ddd278ece701-60d4947f3ccef8f34bb801eb1f9d9aab-1658993832-1658983032

SSC Sports 5
http://iptv.ipslow.com/tv31_www.elahmad.com_ssc_5/index.m3u8?token=5cee1da6f7d660d6268ea58baee57f4c546641f1-24871bd1a76e5a9e527b983310cc6ab1-1658993869-1658983069

SSC Sports 6
http://iptv.ipslow.com/tv31_www.elahmad.com_ssc_6/index.m3u8?token=5fab51fd4cd887dc8f2a36ef65a5d3960403dddc-e318a0c735f57c670d75891e52bf82c4-1658993969-1658983169

SSC Sports 7
https://shls-mbcaction-prod-dub.shahid.net/out/v1/b3ece6f93df54fa39d0065fee8e5d40d/index.m3u8

All Of The m3u8 Links Are From This Website 
http://www.elahmad.com/tv/ssc-sport-live.php

**Comment by brossmee** [comment]
You can watch it from this page:
http://jawaltv.com/srt/

**Comment by freearhey** [comment]
http://iptv.ipslow.com/tv31_www.elahmad.com_ssc_3/index.m3u8?token=fcc78993c540b17f402681987ae609c03d91f9c8-2ddbec0c99c9779c49a8b961581ef8df-1658993768-1658982968 (offline)

**Comment by R4IN2007** [comment]
The links from elahmad changed 
http://www.elahmad.com/tv/ssc-sport-live.php

SSC 1
https://ssc-1-on-prem-ak.akamaized.net/out/v1/ec938957da2849879f0cfac7e309ff38/index.mpd

SSC 2
https://ssc-2-on-prem-ak.akamaized.net/out/v1/d9a2acf5f809461ca47714440fcbc0f4/index.mpd

SSC 3
https://ssc-3-on-prem-ak.akamaized.net/out/v1/ad2b1abef05c419aa6a4e6aee6d269a2/index.mpd

SCC 4 ^ HD
https://ssc-4-on-prem-ak.akamaized.net/out/v1/3e442c7dad1b44b6a0914aa26b62a892/index.mpd

SSC 5
https://ssc-5-on-prem-cw-ak.akamaized.net/out/v1/ee54cd9a3f844ceb8249f430733f9c00/index.mpd

SSC NEWS
https://ssc-news-on-prem-ak.akamaized.net/out/v1/f2c4d3235fc24094a27c2d07ecb3069d/index.mpd

SSC EXTRA 1
https://ssc-extra1-on-prem-cw-ak.akamaized.net/out/v1/d309ac98b822425d9435cca654211498/index.mpd

SSC EXTRA 2
https://ssc-extra2-on-prem-ak.akamaized.net/out/v1/a525c44d152f4d238aa21af75945e29c/index.mpd

SSC EXTRA 3
https://ssc-extra3-on-prem-cw-ak.akamaized.net/out/v1/9d373d000ab64078b131f9d4c18e3502/index.mpd

**Reply by AVGCFan1995** [reply]
Hey @R4IN2007, did you find any livestream links for EBS America since it's hard-to-find nowadays outside of DirectTV.

https://github.com/orgs/iptv-org/discussions/1423

**Reply by Hrishikesh-M-S** [reply]
Any Sky Sports mpd links?

**Reply by karimblock** [reply]
How to play mpd?
ssc sport

**Reply by BellezaEmporium** [reply]
> The links from elahmad changed 
> http://www.elahmad.com/tv/ssc-sport-live.php
> 
> SSC 1
> https://ssc-1-on-prem-ak.akamaized.net/out/v1/ec938957da2849879f0cfac7e309ff38/index.mpd
> 
> SSC 2
> https://ssc-2-on-prem-ak.akamaized.net/out/v1/d9a2acf5f809461ca47714440fcbc0f4/index.mpd
> 
> SSC 3
> https://ssc-3-on-prem-ak.akamaized.net/out/v1/ad2b1abef05c419aa6a4e6aee6d269a2/index.mpd
> 
> SCC 4 ^ HD
> https://ssc-4-on-prem-ak.akamaized.net/out/v1/3e442c7dad1b44b6a0914aa26b62a892/index.mpd
> 
> SSC 5
> https://ssc-5-on-prem-cw-ak.akamaized.net/out/v1/ee54cd9a3f844ceb8249f430733f9c00/index.mpd
> 
> SSC NEWS
> https://ssc-news-on-prem-ak.akamaized.net/out/v1/f2c4d3235fc24094a27c2d07ecb3069d/index.mpd
> 
> SSC EXTRA 1
> https://ssc-extra1-on-prem-cw-ak.akamaized.net/out/v1/d309ac98b822425d9435cca654211498/index.mpd
> 
> SSC EXTRA 2
> https://ssc-extra2-on-prem-ak.akamaized.net/out/v1/a525c44d152f4d238aa21af75945e29c/index.mpd
> 
> SSC EXTRA 3
> https://ssc-extra3-on-prem-cw-ak.akamaized.net/out/v1/9d373d000ab64078b131f9d4c18e3502/index.mpd

In order to watch them you'll either need to get the Widevine licence URL or the Clearkey keys

**Reply by karimblock** [reply]
*snip*

How to play

**Reply by mahmoud30gabr** [reply]
Please have you found a way to run links mpd

**Reply by BellezaEmporium** [reply]
> 
> How to play

You'll need to have a player capable of supporting Widevine streams. 

You either can use Kodi, by using the #KODIPROP tag, specifying the nature of the stream (widevine & key), or use a sort of real-time decryption system (mp4decrypt + ffmpeg should do the trick).

Or you create a website page with dash.js or JWPlayer, by providing the abovementioned clearkey keys.
